apps/dolly-frontend/src/main/js/src/components/fagsystem/tpsf/visning/partials/Nasjonalitet.js

```diff
@@ -75,6 +80,46 @@ export const Nasjonalitet = ({ data, visTittel = true }) => {
 					</DollyFieldArray>
 				</ErrorBoundary>
 			)}
+
+			{pdlPerson?.innflytting?.length > 0 && (
```

Comment:
Blir kanskje litt smør på flesk å vise innvandring og utvandring fra både tpsf og pdl? Så kan sikkert fjerne tpsf? Litt kjedelig for gamle bestillinger, men synes ikke vi kan støtte det for alltid heller 🤔